cache/utils.go

```diff
@@ -0,0 +1,107 @@
+package cache
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"time"
+)
+
+// TODO MOVE
```

Comment:
Still relevant ?

Reply:
It was removed. You must've looked at a specific commit